src/code/z_actor.c

```diff
@@ -4223,7 +4223,7 @@ Gfx D_801AEF88[] = {
     gsSPEndDisplayList(),
 };
 
-Gfx D_801AEFA0[] = {
+Gfx D_801AEFA0[] = { // gActorEmptyDL
```

Comment:
You know? let's rename it!

Reply:
Ah sorry I forgot I added that.

I don't want to rename this in this PR because I think it would be the only change in this PR that would affect other overlays. Even though I have had those kinds of changes in my PRs in the past, I have regretted it and I think it's cleaner to separate wide changes to avoid unexpected conflicts during merges.

Also I don't think it's the best name. It describes the state of the DL (it is empty) but I have seen it used multiple times in different actors and it is always paired with the DL next to it in transparent actors. What I really want to know is what the transparent twin of this DL is doing (and why the non-transparent version doesn't need anything substituted), and what to call that DL to help understand why it is important for transparency but I forgot to ask in discord.